### PR TITLE
Issue756 strobe negative times

### PR DESCRIPTION
**Pull request by bramvdh91**
For #756 

2170cd0 adds some changes that were needed because I kept getting "Unstaged changes" in git, and I could not discard these changes. Solution was to change the line endings in these files.

### CODE REVIEW

**Comment by Mathadon** [comment]
@bramvdh91 2170cd0 removes some files? That does not look like a good idea! Did you run unit tests?

**Comment by Mathadon** [comment]
Also, can you add a revision log entry to the file as is common in the IBPSA library?

**Comment by bramvdh91** [comment]
No unit tests run yet, always takes me a day of debugging to go from my usual Python-Dymola interface set-up before I get it compatible with that of unit testing ;) 

Indeed I was not aware it deleted so many files without replacing them, I thought I had replaced them by files with different line endings. I will revert this and see how to solve it.

**Comment by Mathadon** [comment]
another way of dealing with the line endings is simply to check out those files. That's what I usually do: commit everything that I changed and then do `git checkout \*` to check out the remaining files.

**Comment by bramvdh91** [comment]
@Mathadon The problem was that checking out the files did not help, they kept reappearing, even after a hard reset. 

Made the changes requested, but the unit tests changed a lot of files, probably due to different OS or dymola version. I could rerun these and ignore very small changes in the reference results instead.

**Comment by Mathadon** [comment]
@bramvdh91 I hope to look into this on the relatively short term. I'd first like to set up a computer at TME where everybody can run the unit tests.

**Comment by bramvdh91** [comment]
Great initiative! Will you use a system like the CI of IBPSA?

**Comment by Mathadon** [comment]
Preferably yes, but travis does not allow you to make changes to the actual reference results. So we would also need a system here at TME that allows you to make changes to the reference results. That would not be integrated with GitHub and seems like a larger priority.

**Comment by Mathadon** [comment]
@bramvdh91 we will need to redo this pull request. Did you only make changes to `IDEAS/BoundaryConditions/Occupants/Extern/StrobeInfoManager.mo`? If so, I propose you make a new branch from the master branch, copy the new code into the new branch and then rebuild the pull request from that.  I can help you with running the unit tests!

**Comment by bramvdh91** [comment]
@mathadon I’ll do so if I have a bit more time, currently not at the top of the priority list ;) Expect this in the second half of next week at the soonest.

**Comment by bramvdh91** [comment]
@Mathadon the removed files problem should be solved now, and unit tests can be performed again. 
However, I think a new test to check the new behaviour of the StrobeInfoManager is needed. Could be linked to #745 to clean up this folder.

File: IDEAS/BoundaryConditions/Examples/StrobeInfoManager.mo [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    annotation (Placement(transformation(extent={{-10,-12},{10,8}})));
+  annotation (
+    experiment(StopTime=1e+06),
+    __Dymola_Commands(file=
+          "Resources/Scripts/Dymola/BoundaryConditions/Examples/StrobeInfoManager1.mos"
+        "Unit test 1"),
+    Documentation(revisions="<html>
+<ul>
+<li>
+September 22, 2016 by Filip Jorissen:<br/>
```
Comment (Mathadon) [inline IDEAS/BoundaryConditions/Examples/StrobeInfoManager.mo]: @bramvdh91 you forgot to edit the revision history here. I'll fix this!

File: IDEAS/BoundaryConditions/Examples/StrobeInfoManager_offset.mo [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    annotation (Placement(transformation(extent={{-10,-12},{10,8}})));
+  annotation (
+    experiment(StopTime=1e+06),
+    __Dymola_Commands(file=
+          "Resources/Scripts/Dymola/BoundaryConditions/Examples/StrobeInfoManager_offset.mos"
+        "Unit test 1"),
+    Documentation(revisions="<html>
+<ul>
+<li>
+September 22, 2016 by Filip Jorissen:<br/>
```
Comment (Mathadon) [inline IDEAS/BoundaryConditions/Examples/StrobeInfoManager_offset.mo]: revision history

**Pull request merged**